.circleci/config.yml

```diff
@@ -88,262 +89,264 @@ jobs:
+            fi
   frontend_deploy:
     working_directory: /home/circleci/tasking-manager
+    resource_class: large
     docker:
-    - image: cimg/python:3.7.14-node
+      - image: cimg/python:3.7.14-node
     parameters:
       stack_name:
         description: "the name of the stack for cfn-config"
         type: string
     steps:
-    - checkout
-    - setup_remote_docker
-    - run:
-        name: Install modules
-        command: |
-          pip install awscli --upgrade
-    - run:
-        name: Configure AWS Access Key ID
-        command: |
-          aws configure set aws_access_key_id \
-          $AWS_ACCESS_KEY_ID \
-          --profile default
-    - run:
-        name: Configure AWS Secret Access Key
-        command: |
-          aws configure set aws_secret_access_key \
-          $AWS_SECRET_ACCESS_KEY \
-          --profile default
-    - run:
-        name: Configure AWS default region
-        command: |
-          aws configure set region $AWS_REGION \
-          --profile default
-    - run:
-        name: Deploy Frontend to S3
-        command: |
-          cd ${CIRCLE_WORKING_DIRECTORY}/frontend/
-          export TM_ENVIRONMENT=<< parameters.stack_name >>
-          yarn
-          CI=true GENERATE_SOURCEMAP=false yarn build
-          aws s3 sync build/ s3://tasking-manager-<< parameters.stack_name >>-react-app --delete --cache-control max-age=31536000
-          aws s3 cp s3://tasking-manager-<< parameters.stack_name >>-react-app s3://tasking-manager-<< parameters.stack_name >>-react-app --recursive --exclude "*" --include "*.html" --metadata-directive REPLACE --cache-control no-cache --content-type text/html
-          export DISTRIBUTION_ID=`aws cloudformation list-exports --output=text --query "Exports[?Name=='tasking-manager-<< parameters.stack_name >>-cloudfront-id-${AWS_REGION}'].Value"`
-          aws cloudfront create-invalidation --distribution-id $DISTRIBUTION_ID --paths "/*"
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install modules
+          command: |
+            pip install awscli --upgrade
+      - run:
+          name: Configure AWS Access Key ID
+          command: |
+            aws configure set aws_access_key_id \
+            $AWS_ACCESS_KEY_ID \
+            --profile default
+      - run:
+          name: Configure AWS Secret Access Key
+          command: |
+            aws configure set aws_secret_access_key \
+            $AWS_SECRET_ACCESS_KEY \
+            --profile default
+      - run:
+          name: Configure AWS default region
+          command: |
+            aws configure set region $AWS_REGION \
+            --profile default
+      - run:
+          name: Deploy Frontend to S3
+          command: |
+            cd ${CIRCLE_WORKING_DIRECTORY}/frontend/
+            export TM_ENVIRONMENT=<< parameters.stack_name >>
+            yarn
```

Comment:
We should probably use the cache here instead of rebuilding.. maybe there is a risk of using the wrong cached folder though...